aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -17,6 +22,48 @@ namespace {
 #endif
 }
 
+static const std::vector<DispatchKey>& allDispatchKeysInFullSet() {
+  static const auto result = ([]() {
+    std::vector<DispatchKey> vec;
+    for (const auto dispatch_key: DispatchKeySet(DispatchKeySet::FULL)) {
+      vec.push_back(dispatch_key);
+    }
+    return vec;
+  })();
```

Comment:
Nit (or point me to some doc that saying that extra parenthesis are good for lambda definitions 
```suggestion
  static const auto result = []() {
    std::vector<DispatchKey> vec;
    for (const auto dispatch_key: DispatchKeySet(DispatchKeySet::FULL)) {
      vec.push_back(dispatch_key);
    }
    return vec;
  }();
```